api/rest/cluster_bundle_handler.go

```diff
@@ -281,19 +281,21 @@ func (c *ClusterBundleHandler) Download(w http.ResponseWriter, r *http.Request)
 	var masterWithBundle node
 	found := false
 	for _, n := range masters {
-		_, err = c.client.Status(ctx, n.baseURL, id)
-		if err != nil {
-			switch err.(type) {
+		bundle, e := c.client.Status(ctx, n.baseURL, id)
```

Comment:
Is there a specific reason to rename the `err` variable? I'd say it's convenience across our projects to call it `err`.

Reply:
```api/rest/cluster_bundle_handler.go:284:11: shadow: declaration of "err" shadows declaration at line 273 (govet)
		bundle, err := c.client.Status(ctx, n.baseURL, id)
```

Reply:
I think it's better to change name than disable linter

Reply:
Then I'd prefer sth. more meaningful such as `statusErr`.

Reply:
Fixed